src/main/java/cms/gov/madie/measure/services/ExportService.java

```diff
@@ -42,4 +42,14 @@ private void validateMetadata(Measure measure) {
       }
     }
   }
+
+  public byte[] getQRDA(Measure measure, String accessToken) {
+    validateMetadata(measure);
+    ModelValidator modelValidator =
+        modelValidatorFactory.getModelValidator(ModelType.valueOfName(measure.getModel()));
+    modelValidator.validateGroups(measure);
```

Comment:
this could be unnecessary because the pre-requisite is to run the test cases and we do not run test cases for invalid measures

Reply:
@adongare So I should remove the validation right?

Reply:
@adongare If I remove the validation, could it cause problems when calling the api directly from postman? I think QA test apis directly

Reply:
good point, it would. let's leave it then.